raiden/tests/utils/events.py

```diff
@@ -1,15 +1,17 @@
 from collections.abc import Mapping
+from typing import Any, Iterable, List, Optional, Tuple, Type, TypeVar, cast
 
 import gevent
 
 from raiden.raiden_service import RaidenService
 from raiden.storage.sqlite import RANGE_ALL_STATE_CHANGES
 from raiden.transfer.architecture import Event, StateChange
 from raiden.transfer.mediated_transfer.events import EventUnlockClaimFailed, EventUnlockFailed
-from raiden.utils.typing import Any, Iterable, List, Optional, Tuple, Type, TypeVar
 
 NOVALUE = object()
 T = TypeVar("T")
+T_Event = TypeVar("T_Event", bound="Event")
```

Comment:
Not sure how this should be named, I think we use`T_*` usually for the underlying type. Also in line 15 a non `T_` prefixed name is used.

Reply:
For the `StateChange` we already have one defined below, called `SC`. I should remove that line. I could rename `T_Event` to `TE`, which would be in line with `TM` below. Not the best names, but at least consistent.

Reply:
Sounds good to me.